src/server/scripts/Kalimdor/CavernsOfTime/BattleForMountHyjal/hyjal_trash.cpp

```diff
@@ -1172,12 +1194,14 @@ class npc_frost_wyrm : public CreatureScript
             Initialize();
             instance = creature->GetInstanceScript();
             go = false;
+            me->SetCanFly(true);
```

Comment:
It shouldn't be called in scripts, also those SetDisableGravity wont work like this without extra_flags 512 (it only 'works' currently because of SetCanFly, but if removed, they'll stop flying)
They just should have creature_template_movement with Flight = 1, same with npc_gargoyle

Reply:
Add creature_template_movement with Flight = 1 and remove SetCanFly in script will work correctly?

Reply:
It works.